Core/HLE/sceMpeg.cpp

```diff
@@ -1495,6 +1496,9 @@ void PostPutAction::run(MipsCall &call) {
 // Program signals that it has written data to the ringbuffer and gets a callback ?
 static u32 sceMpegRingbufferPut(u32 ringbufferAddr, int numPackets, int available)
 {
+	// Generally, program will call sceMpegRingbufferAvailableSize() before this func.
+	// Still need to check available?
```

Comment:
See #978.  I think we should test this, though, for the correct behavior.  Ideally we should replicate (as far as is visible) what firmware does.

-[Unknown]